resources/js/beatmap-discussions/review-editor-config-context.ts

```diff
@@ -4,4 +4,4 @@
 import ReviewEditorConfigJson from 'interfaces/review-editor-config-json';
 import * as React from 'react';
 
-export const ReviewEditorConfigContext = React.createContext({} as ReviewEditorConfigJson);
+export const ReviewEditorConfigContext = React.createContext<ReviewEditorConfigJson>({ max_blocks: 0 });
```

Comment:
what's this change doing here 🤔 although I guess it's simple enough so this can stay

Reply:
`max_blocks` isn't supposed to be `undefined` and this just makes the default 0 so pages that don't need editing can skip adding a provider without causing `max_blocks` to be `undefined`